### DISCUSSION TITLE
how to allow upload files?

### DISCUSSION THREAD

**Discussion by Misagh90** [spatie/laravel-html#200] [Q&A] [answered]
Hi,
I create a form like this:
`{{ html()->form('post', '/admin/entity')->open()->acceptsFiles() }}`
but I have an error: "Call to undefined method Illuminate\Support\HtmlString::acceptsFiles()"

how to allow files for forms?

**Comment by Misagh90** [comment, marked as answer]
I fix it by adding "acceptsFiles()" before "open"  like this:
`{{ html()->form('POST')->route('entity.store')->acceptsFiles()->open() }}`

**Reply by airmjay** [reply]
Thanks sir.
